authz/policy/drools/src/main/java/org/keycloak/authorization/policy/provider/drools/DroolsPolicyProviderFactory.java

```diff
@@ -150,4 +158,10 @@ void remove(Policy policy) {
     KieContainer getKieContainer(String groupId, String artifactId, String version) {
         return this.ks.newKieContainer(this.ks.newReleaseId(groupId, artifactId, version));
     }
+
+    private void checkIfFeatureIsEnabled() {
```

Comment:
Would it not be better to use the approach used by other providers:
https://github.com/keycloak/keycloak/blob/master/services/src/main/java/org/keycloak/authentication/authenticators/browser/ScriptBasedAuthenticatorFactory.java#L155